faq/faq_central.php

```diff
@@ -2,323 +2,5 @@
 $relPath = '../pinc/';
```

Comment:
We should just delete this file entirely. Not surprisingly, it hasn't shown up in our web logs in a Very Long Time because FAQ / help links for this go directly to the wiki and not here.